### DISCUSSION TITLE
Smart chat conversation with notes based on tags

### DISCUSSION THREAD

**Discussion by migui3230** [brianpetro/obsidian-smart-connections#271] [Q&A]
@brianpetro Is the model able to process tags and give me answers if I ask it questions about my notes with a specific tag?

**Example:**

Based on my notes with the tag of Twitter, what information do you have when it comes to X?

**Comment by brianpetro** [comment]
@migui3230 

Currently, no. You cannot use tags in the Smart Chat as you described.

However, this would certainly be useful. I labelled this discussion as an "enhancement" for development as a future feature.

Thanks for the question!

Brian 🌴
